controllers/deactivation/mapper.go

```diff
@@ -0,0 +1,38 @@
+package deactivation
+
+import (
+	"errors"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type UserSignupToMasterUserRecordMapper struct {
+	client client.Client
+}
+
+var _ handler.Mapper = UserSignupToMasterUserRecordMapper{}
+var mapperLog = ctrl.Log.WithName("UserSignupToMasterUserRecordMapper")
+
+func (b UserSignupToMasterUserRecordMapper) Map(obj handler.MapObject) []reconcile.Request {
+	if userSignup, ok := obj.Object.(*toolchainv1alpha1.UserSignup); ok {
+
+		req := []reconcile.Request{}
+
+		req = append(req, reconcile.Request{
+			NamespacedName: types.NamespacedName{Namespace: userSignup.Namespace, Name: userSignup.Status.CompliantUsername},
```

Comment:
`CompliantUsername` can be empty. This should be fine but maybe we want to return an empty `[]Request{}` in this case?